lucien-ios/Dashboard/DashboardComicCells/CollectionViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LendingCollectionViewCell.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/10/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class CollectionViewCell: UICollectionViewCell {
```

Comment:
Let's rename this to something more specific - `DashboardCollectionViewCell` would be better